platform-tests/src/test/java/org/junit/platform/launcher/core/DefaultLauncherTests.java

```diff
@@ -1019,7 +1019,7 @@ public TestDescriptor discover(EngineDiscoveryRequest discoveryRequest, UniqueId
 		inOrder.verify(executionListener).executionStarted(
 			argThat(d -> d.getUniqueIdObject().equals(UniqueId.forEngine("engine-id"))));
 		inOrder.verify(executionListener).executionSkipped(
-			argThat(d -> d.getUniqueIdObject().getLastSegment().getType().equals("container")),
+			argThat(d -> "container".equals(d.getUniqueIdObject().getLastSegment().getType())),
```

Comment:
check seems inconsistent as already approached in:

- https://github.com/junit-team/junit-framework/pull/4748
- https://github.com/junit-team/junit-framework/commit/aca30c5b64805a424b6639ec9ceecc23a5cd5c10